R/ospsuite.utils-env.R

```diff
@@ -0,0 +1,45 @@
+# Environment that holds various global variables and settings for the ospsuite.utils,
+# It is not exported and should not be directly manipulated by other packages.
+ospsuiteUtilsEnv <- new.env(parent = emptyenv())
+
+# name of the package. This will be used to retrieve information on the package at run time
+ospsuiteUtilsEnv$packageName <- "ospsuite.utils"
+
+ospsuiteUtilsEnv$suiteName <- "Open Systems Pharmacology"
+
+# Separator defined in OSPSuite.Core.
+ospsuiteUtilsEnv$pathSeparator <- "|"
+
+# Default values for the formatNumerics() helper function
+ospsuiteUtilsEnv$formatNumericsDigits <- 2
```

Comment:
This was 5 in ospsuite, but 2 as magic number of `formatNumerics` here.. dunno whats better.

Reply:
There is no better number, it really depends on the application.
Here, `2` was defined as default because the function is usually called to format displayed data.frames so more than `2` makes them over-crowded.
The default from `ospsuite` package is maybe used for a different display and was actually combined with the parameter `nsmall` which I think was `2` causing the function `format` to display 2 digits only.
```r
> format(1, digits = 5, nsmall = 2)
[1] "1.00"
```